assets/src/js/search.js

```diff
@@ -8,6 +8,9 @@ if (!window.lazyLoad) {
 
 // Search page.
 export const setupSearch = function($) {
+  const isSearch = !!$('body.search').length;
+  if (!isSearch) return;
```

Comment:
Need to add curly brackets here, eslint is complaining about it -
`12:3  error  Expected { after 'if' condition  curly`

Reply:
@sagarsdeshmukh fixed!